### DISCUSSION TITLE
Version 2.13.0-beta2

### DISCUSSION THREAD

**Discussion by AndrewDavidoff** [OfficeDev/Open-XML-SDK#921] [General]
### Added
- Additional O19 types to match Open Specifications (#916)

<hr /><em>This discussion was created from the release <a href='https://github.com/OfficeDev/Open-XML-SDK/releases/tag/v2.13.0-beta2'>Version 2.13.0-beta2</a>.</em>

**Comment by the-frank** [comment]
On the issues section, I found a code sample to read "ThreadedComments", but I'm still trying to figure out how to add one programmatically and attach it to a cell?

I gladly will take a look of any code sample doing that if it exists (but could not find it so far).
Thanks
Francis
